.gitignore

```diff
@@ -8,6 +8,8 @@ sw.js.map
 package-lock.json
 /build/
 /coverage/
+/docs/_build/
+/docs/locales/**/*.mo
```

Comment:
From what I can tell, `*.mo` files are compiled translation units, and should not be kept in Git history. `*.po` files are the translation files that will actually be tracked in the future.